packages/web-components/fast-ssr/src/styles/fast-style.fixture.html

```diff
@@ -388,11 +388,11 @@ <h2>Heading</h2>
                 </div>
             </template>
         </fast-card>
-        <fast-card style="width: 300px;">
+        <fast-card style="width: 300px">
             <template shadowroot="open">
                 <fast-style style-id="fast-card"></fast-style>
                 <img src="https://via.placeholder.com/300x200/414141" />
-                <div style="padding: 0 10px 10px; color: var(--neutral-foreground-rest);">
+                <div style="padding: 0 10px 10px; color: var(--neutral-foreground-rest)">
```

Comment:
Are we sure about not having the trailing semi-colon?

Reply:
I support the trailing semicolon as it's how I personally write manual css and it aligns with other language decisions like maintaining a trailing comma or even the semicolon in JS which isn't always strictly required. Is there an option to turn it on?

Reply:
We are opted into the only configuration for semi colons. This is prescriptive on Prettier's part and not something we can adjust with current configuration. On the plus side, you can continue to use these and prettier will take care of removing as necessary. If they change to include it, we can update and it'll include it.

Reply:
Hmm, maybe the trailing comma is gone, at least it wasn't added in some places I would have expected it. I don't have a strong opinion on this though (aside from still not agreeing with Prettier's model in general).

Reply:
My style has generally been to not include the trailing SC, but I don't think there is a right or wrong here so whatever people want is cool with me.

Reply:
We don’t have a choice so folks can leave it in or remove but prettier is gonna do what it wants :)